lib/manager/bundler/artifacts.ts

```diff
@@ -18,12 +17,63 @@ import {
 } from '../../constants/error-messages';
 import { BinarySource } from '../../util/exec/common';
 
-export async function updateArtifacts({
-  packageFileName,
-  updatedDeps,
-  newPackageFileContent,
-  config,
-}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+async function getDockerTag(updateArtifact: UpdateArtifact): Promise<string> {
```

Comment:
```typescript
interface DockerOptions {
  ...
  tag?: string | (UpdateArtifact) => Promise<string>;
  ...
}
```

?

Reply:
But we can't provide `UpdateArtifact` argument from `dockerCmd(...)` call, so should be `(void) => Promise<string>` actually

Reply:
Let's calculate the tag within this file and then pass it through to util. We don't need to do the tag lookup in util yet.